<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchLabelDialog</class>
 <widget class="QDialog" name="BatchLabelDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>720</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Batch Labeler</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <family>Adobe Devanagari</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="tabBarAutoHide" stdset="0">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>XY</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>2</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>2</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>2</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2">
        <widget class="QGraphicsView" name="graphicsView_1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>2</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>YZ</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="2">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>2</width>
           <height>2</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>2</width>
           <height>2</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <widget class="QGraphicsView" name="graphicsView_2"/>
       </item>
       <item row="2" column="3">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>2</width>
           <height>2</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>2</width>
           <height>2</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>XZ</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="2">
        <widget class="QGraphicsView" name="graphicsView_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>2</width>
           <height>2</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="3">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>2</width>
           <height>2</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>2</width>
           <height>2</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>2</width>
           <height>2</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>420</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Sample List</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="margin">
          <number>15</number>
         </property>
         <item>
          <widget class="QListView" name="listView">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QRadioButton" name="radioButton_0">
            <property name="font">
             <font>
              <family>SimSun-ExtB</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Level 0</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="radioButton_5">
            <property name="font">
             <font>
              <family>SimSun-ExtB</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Level 5</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="radioButton_1">
            <property name="font">
             <font>
              <family>SimSun-ExtB</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Level 1</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QRadioButton" name="radioButton_6">
            <property name="font">
             <font>
              <family>SimSun-ExtB</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Level 6</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="radioButton_2">
            <property name="font">
             <font>
              <family>SimSun-ExtB</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Level 2</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QRadioButton" name="radioButton_7">
            <property name="font">
             <font>
              <family>SimSun-ExtB</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Level 7</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="radioButton_3">
            <property name="font">
             <font>
              <family>SimSun-ExtB</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Level 3</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QRadioButton" name="radioButton_8">
            <property name="font">
             <font>
              <family>SimSun-ExtB</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Level 8</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QRadioButton" name="radioButton_4">
            <property name="font">
             <font>
              <family>SimSun-ExtB</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Level 4</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QRadioButton" name="radioButton_9">
            <property name="font">
             <font>
              <family>SimSun-ExtB</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Level 9</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QPushButton" name="button_previous">
            <property name="font">
             <font>
              <family>Bahnschrift</family>
              <pointsize>11</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Previous</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_next">
            <property name="font">
             <font>
              <family>Bahnschrift</family>
              <pointsize>11</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Next</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_remove">
            <property name="font">
             <font>
              <family>Bahnschrift</family>
              <pointsize>11</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_v3d">
            <property name="font">
             <font>
              <family>Bahnschrift</family>
              <pointsize>11</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>View in Vaa3D</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_save">
            <property name="font">
             <font>
              <family>Bahnschrift</family>
              <pointsize>11</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listView</tabstop>
  <tabstop>radioButton_0</tabstop>
  <tabstop>radioButton_1</tabstop>
  <tabstop>radioButton_2</tabstop>
  <tabstop>radioButton_3</tabstop>
  <tabstop>radioButton_4</tabstop>
  <tabstop>radioButton_5</tabstop>
  <tabstop>radioButton_6</tabstop>
  <tabstop>radioButton_7</tabstop>
  <tabstop>radioButton_8</tabstop>
  <tabstop>radioButton_9</tabstop>
  <tabstop>button_previous</tabstop>
  <tabstop>button_next</tabstop>
  <tabstop>button_remove</tabstop>
  <tabstop>button_v3d</tabstop>
  <tabstop>button_save</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>graphicsView_3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
